sentiment_analysis_model.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+"""
+Spyder Editor
+
+"""
+
+import numpy as np 
+import pandas as pd 
+from sklearn.model_selection import train_test_split 
+
+import nltk
+from nltk.corpus import stopwords
+from nltk.classify import SklearnClassifier
+
+from wordcloud import wordcloud,STOPWORDS
+
+from subprocess import check_output
+
+import mlflow 
+
+
+class Sentiment:
+   def __init__(self):
+    
+       data = pd.read_csv(r'C:\Users\user\Downloads\Sentiment.csv')
+
+       mlflow.log_param('data',data)

```

Comment:
Please register parameters in MLproject file. Are you trying to add csv file name as parameter? Then please update it appropriately.